tox.ini

```diff
@@ -10,18 +10,13 @@ basepython =
 
 [testenv]
 install_command = pip install -e ".[test]" -U {opts} {packages}
-commands = pytest {posargs}
+commands = pytest --cov wagtail_live {posargs}
 extras = test
 deps =
-    pytest-django>=4.3.0,<5
```

Comment:
These dependencies are already installed a few lines above this by `pip install -e ".[test]"`.

Have removed them here to avoid future dependency conflicts.